hack/verify-dockerless.sh

```diff
@@ -0,0 +1,15 @@
+not_have_dockerless=$(find ../pkg/kubelet/dockershim -name "*.go" ! -exec grep -q 'dockerless' {} \; -print)
+if [[ $not_have_dockerless ]]; then
+        echo "Following files do not contain dockerless tag"
```

Comment:
Since these checks are running as part of CI, manually outputting "violations" isn't sufficient. We also want the script to fail.

Specifically, for this condition statement, we want to check that no files are found.